### PR TITLE
Feature/network requests

### PR DESCRIPTION
**Pull request by RMartin0717**
# Pull Request Template
## Description

Worked as a whole team to set up get requests in scripts file to get user, recipe, and ingredients data from API

Fixes # (issue)
## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -10,27 +12,85 @@ import Recipe from './recipe';
...
+//     console.log("all user data", allUsersData)
+// }
+
+function onStartup(allUsersData, allIngredientsData, allRecipeData) {
+  createRandomUser(allUsersData);
+  createUserPantry(allIngredientsData);
+  createRecipeRepo(allRecipesData);
+  //do something allIngredientsData
+  console.log("users data", allUsersData)
+  console.log("ingredients data", allIngredientsData)
```
Comment (bhohnco) [inline src/scripts.js:75]: We'll need to take these out later but looks good!

**Review by bhohnco — APPROVED** [review]

File: src/scripts.js [hunk 2]
```diff
@@ -10,27 +12,85 @@ import Recipe from './recipe';
...
 
-window.onload = onStartup();
+
+Promise.all([retrieveUserData, retrieveIngredientsData, retrieveRecipesData])
+  .then((data) => {
+    // console.log(data)
+    allUsersData = data[0]
+    allIngredientsData = data[1]
+    allRecipesData = data[2]
+    onStartup(allUsersData, allIngredientsData, allRecipesData);
```
Comment (michannstoner) [inline src/scripts.js:46]: Nice work getting this data stored by putting this function inside of the promsise.

File: src/scripts.js [hunk 3]
```diff
@@ -10,27 +12,85 @@ import Recipe from './recipe';
...
+//   createRandomUser();
+//   populateCards(recipeRepository.recipes);
+//   greetUser();
+//     console.log("all user data", allUsersData)
+// }
+
+function onStartup(allUsersData, allIngredientsData, allRecipeData) {
+  createRandomUser(allUsersData);
+  createUserPantry(allIngredientsData);
+  createRecipeRepo(allRecipesData);
```
Comment (michannstoner) [inline src/scripts.js:72]: Appreciate creating individual functions for each of these rather than building all of it in the onStartup function

**Review by michannstoner — APPROVED** [review]
Nice work on the beginning of pulling in data using fetch!

**Pull request merged**